misc/backfill_qc_states.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+
+import numpy
+import pandas
+
+from lang_qc.db.helper.well import WellMetrics, WellQc
+from lang_qc.db.mlwh_connection import get_mlwh_db
+from lang_qc.db.qc_connection import get_qc_db
+from lang_qc.util.auth import get_user
+
+
+def _check_df_shape(df):
+    print(df.shape)
+    assert df.empty is False, "Dataframe is empty"
```

Comment:
Assertions can be disabled as part of `python -o`, beware of trusting them outside of tests. It's probably ok for this sort of hackery, but not for embedding into web servers.